src/openklant/utils/tests/test_validators.py

```diff
@@ -83,6 +83,8 @@ def test_validate_phone_number(self):
             "1400",
             "14012",
             "14079",
+            "0313028612",
```

Comment:
Unrelated to your changes, but could you change this `ValidatorsTestCase` to inherit from `from django.test.SimpleTestCase` instead of `TestCase`? `TestCase` also sets up the database and runs migrations, while `SimpleTestCase` doesn't, so the latter is faster if you don't use the database in your tests (which is the case for this file).